src/Symfony/Component/CssSelector/Node/ClassNode.php

```diff
@@ -38,7 +38,7 @@ class ClassNode extends AbstractNode
     public function __construct(NodeInterface $selector, $name)
     {
         $this->selector = $selector;
-        $this->name = $name;
+        $this->name = str_replace('\\', '', $name);
```

Comment:
What `:` as mentioned in the ticket. Can we find the related part of the spec talking about that just to be sure that we are covering all possibilities here?